lib/mayaUsd/render/vp2ShaderFragments/shaderFragments.h

```diff
@@ -29,6 +31,7 @@ class HdVP2ShaderFragments
 {
 public:
     //! Register all HdVP2 fragments
+    MAYAUSD_CORE_PUBLIC
     static MStatus registerFragments();
```

Comment:
The UsdPreviewSurfaceShadingNodeOverride can also request for registerFragments( )